src/karma/KarmaParser.spec.ts

```diff
@@ -93,6 +93,15 @@ describe("KarmaParser", () => {
             expect(karmaRequests.length).to.equal(1);
             expect(karmaRequests[0]).to.eql(expectedRequest);
         });
+
+        it("handles the user reference strings correctly", () => {
```

Comment:
Nitpicking - I'd prefer this to be more descriptive; as-is, it's not clear what "correctly" means. 

Something like it "wraps user reference strings in < > symbols"